nxdrive/engine/engine.py

```diff
@@ -9,6 +9,7 @@
 from typing import Any, Callable, Dict, List, Optional, Type, TYPE_CHECKING
 from urllib.parse import urlsplit
 
+import requests
```

Comment:
Maybe it is worth replacing this import in `_check_https()` only when we need it?